src/wallet/api.rs

```diff
@@ -1188,14 +1199,12 @@ impl Wallet {
         feerate: f64,
         manually_selected_outpoints: Option<Vec<OutPoint>>,
     ) -> Result<Vec<(ListUnspentResultEntry, UTXOSpendInfo)>, WalletError> {
-        log::info!(
-            "Coinselect Target : {} Sats with No. of Manually Selected Utxos: {}",
-            amount.to_sat(),
-            manually_selected_outpoints
-                .as_deref()
-                .unwrap_or_default()
-                .len()
-        );
+        if manually_selected_outpoints.is_some() {
+            log::debug!(
```

Comment:
Debug logs should be exclusively used to prinbt internal data structs and info required for debug. We don't want any debug log at this stage.

This can be either info, or can be removed. I think keeping it in info is fine.